pyvista/utilities/helpers.py

```diff
@@ -215,12 +215,16 @@ def point_array(obj, name):
 
     Returns
     -------
-    numpy.ndarray
+    pyvista.pyvista_ndarray
         Wrapped array.
 
     """
-    vtkarr = obj.GetPointData().GetAbstractArray(name)
-    return convert_array(vtkarr)
+    if isinstance(obj, pyvista.DataSet):
+        try:
+            return obj.point_data.get_array(name)
+        except KeyError:  # pragma: no cover
+            return None
+    return pyvista.pyvista_ndarray(obj.GetPointData().GetAbstractArray(name))
 
 
 def field_array(obj, name):
```

Comment:
Looking at this duplication in the original code and the this PR, how bad would it be to define an internal helper that does something like
```py
def _assoc_array(obj, name, association='point'):
    VTK_attr = f'Get{association.title()}Data'
    python_attr = f'{association.lower()}_data'
    if isinstance(obj, pyvista.DataSet):
        return getattr(obj, python_attr).get(name)
    return pyvista.pyvista_ndarray(getattr(obj, VTK_attr)().GetAbstractArray(name))
```
(potentially with more temporary variables for readability) and then making `point_array`, `field_array` and `cell_array` a thin wrapper for that?

Reply:
Agree with [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself). Refactored in 1d155ca710ef906ec7bb2554f3a5c620b6a0e05d.